data/human/free worlds 3 reconciliation.txt

```diff
@@ -1320,6 +1320,19 @@ mission "FW Pug 2C: A Jump Drive"
 
 
 
+mission "FW Pug Jump Drive Check"
+	landing
+	to offer
+		has "fw given jump drive"
+		not "event: pug territory liberated"
```

Comment:
Aren't there links made into the Pug occupied systems before this point, though? That is, the player could lose their JD after being given it but not need it anymore before this event.

Reply:
The story expects you to use a JD until the final push with the republic but you dont need one after opening the first syndicate link really